.gitignore

```diff
@@ -7,6 +7,7 @@
 
 # testing
 /coverage
+/cypress
```

Comment:
Are you sure you can ignore the whole cypress folder? If it contains any of your tests you need to keep it in the repo so other coders can run the tests.

Reply:
I'll add the folder again next week once I hopefully know more about writing tests that make sense. 😄